src/pages/ShoppingListPage.re

```diff
@@ -17,7 +17,9 @@ type action =
   | ShoppingListFetchSuccess(shoppingList)
   | ApiCallError(string)
   | AddShoppingItem
-  | ChangeShoppingItem(int, shoppingItem);
+  | ChangeShoppingItem(int, shoppingItem)
+  | SetListStatus(listStatus)
+  | ToggleShoppingItem(int);
```

Comment:
Maybe `ToggleShoppingItemPurchased`? Or a name that would have `Purchased` in it so that it is clear that you are toggling that property of an item. What do you think?

Reply:
Fixed.